### PR TITLE
New Ms records Abb 107 and 270

### PR DESCRIPTION
**Pull request by nafisa-valieva**
Hello! Here I have created two records for Abb107 and 270, needed for Carsten's work.

### CODE REVIEW

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Comment by CarstenHoffmannMarburg** [comment]
Thank you!

File: ParisBNF/abb/BNFabb107.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                    <term key="ChristianLiterature"/>
+                    <term key="BetaEsraelLiterature"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="NV" when="2024-10-24">Created a record.</change>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb107.xml:86]: I would write "Created a stub" to make it clear that this is supposed to be developed to a record later

**Review by karljonaskarlsson — APPROVED** [review]

File: ParisBNF/abb/BNFabb107.xml (hunk 1 above)
Comment (nafisa-valieva) [reply ParisBNF/abb/BNFabb107.xml:86]: Ok! I will change it!

**Review by karljonaskarlsson — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Comment by eu-genia** [comment]
Were these changes merged in another pull request? Or why was this closed without merge?

**Comment by eu-genia** [comment]
@nafisa-valieva in this branch you had created two new records 
image

the stubs have been reviewed and approved.

what is the reason behind your decision to close this pull request without merging and delete your work? (you probably have a reason but it is not obvious to me). Thank you.

**Comment by nafisa-valieva** [comment]
> @nafisa-valieva in this branch you had created two new records image
> 
> the stubs have been reviewed and approved.
> 
> what is the reason behind your decision to close this pull request without merging and delete your work? (you probably have a reason but it is not obvious to me). Thank you.

I wanted to delete all the old branches I had...  If you can restore the branch, I will check once again to make sure all changes were merged! I am sorry!

**Pull request merged**